src/main/java/com/powsybl/openloadflow/network/impl/HvdcConverterStations.java

```diff
@@ -45,27 +44,31 @@ public static boolean isVsc(Identifiable<?> identifiable) {
                 && ((HvdcConverterStation<?>) identifiable).getHvdcType() == HvdcConverterStation.HvdcType.VSC;
     }
 
-    public static boolean isHvdcDanglingInIidm(HvdcConverterStation<?> station, LfNetworkParameters parameters) {
+    public static boolean isHvdcDanglingInIidm(HvdcConverterStation<?> station) {
 
-        if (isIsolated(station.getTerminal(), parameters)) {
+        if (isIsolated(station.getTerminal())) {
             return true;
         } else {
             return station.getOtherConverterStation().map(otherConverterStation -> {
                 Terminal otherTerminal = otherConverterStation.getTerminal();
-                return isIsolated(otherTerminal, parameters);
+                return isIsolated(otherTerminal);
             }).orElse(true); // it means there is no HVDC line connected to station
         }
     }
 
-    private static boolean isIsolated(Terminal terminal, LfNetworkParameters parameters) {
-        Bus bus = parameters.isBreakers() ? terminal.getBusBreakerView().getBus() : terminal.getBusView().getBus();
+    private static boolean isIsolated(Terminal terminal) {
+        Bus bus = terminal.getBusView().getBus();
         if (bus == null) {
             return true;
         }
 
-        // The criteria should as close as possible to Networks.isIsolatedBusForHvdc - only connected to the station
+        // The criteria should as close as possible to Networks.isIsolatedBusForHvdc - only connected to the station or a fictitious load
         return bus.getConnectedTerminalStream()
                 .map(Terminal::getConnectable)
-                .noneMatch(c -> !(c instanceof HvdcConverterStation<?> || c instanceof BusbarSection));
+                .noneMatch(c -> !(c instanceof HvdcConverterStation<?> || c instanceof BusbarSection || isFictitiousLoad(c)));
```

Comment:
Just for readability : Can we remove the double negation ?

`return !bus.getConnectedTerminalStream().map(Terminal::getConnectable).anyMatch(c -> c instanceof HvdcConverterStation<?> || c instanceof BusbarSection || isFictitiousLoad(c));`

Reply:
Your formula is not equivalent. You want to find nothing else than a station, a busbar or a fictive load.
But your statement says that if their is no station (which never happens by the way) or no busbar or no fictive load, the bus is isolated.

Reply:
Sorry, you are right indeed :+1: